client/reader/recommendations/posts/index.jsx

```diff
@@ -0,0 +1,26 @@
+var React = require( 'react' );
```

Comment:
This file should probably use ES6. Also the namespace is not quite right. It should be `posts-recommendations` or similar but not just `posts`.